### ISSUE TITLE
Regex with escaped characters in response body requires dobule escaping

### ISSUE THREAD

**Issue by mariuszs** [Codearte/accurest#126]

**Comment by mariuszs** [comment]
Sample test -> #120 

This also make unusable alias like email() in response body.

**Comment by marcingrzejszczak** [comment]
Should be fixed in 0.9.0. Can you confirm that?

**Comment by mariuszs** [comment]
```
response {
    status 200
    body("""{"property":"${value(client('123'), server(regex('\\d+')))}"}""")
    }
```
